Python/Product/PythonTools/PythonTools/Navigation/DropDownBarClient.cs

```diff
@@ -72,9 +74,10 @@ class DropDownBarClient : IVsDropdownBarClient {
 
             _serviceProvider = serviceProvider;
             _uiThread = _serviceProvider.GetUIThread();
+            _services = _serviceProvider.GetComponentModel().GetService<PythonEditorServices>();
             _analysisEntry = analysisEntry;
-            textView.TextBuffer.RegisterForParseTree(ParserOnNewParseTree);
```

Comment:
Did these extension methods go away?  I don't see a delete of them anywhere but it seems like the callers are all disappearing?